chain/chain/src/chain.rs

```diff
@@ -1064,45 +1064,10 @@ impl Chain {
     /// upgrade.
     pub fn verify_challenges(
```

Comment:
Can we remove this completely?

Reply:
I intentionally stopped removing code at this point to keep the size of the PR from exploding.  I will be removing this function in future PRs.